mailbox/elasticsearch/pom.xml

```diff
@@ -34,11 +34,11 @@
     <dependencies>
         <dependency>
             <groupId>${james.groupId}</groupId>
-            <artifactId>apache-james-backends-es-v7</artifactId>
+            <artifactId>apache-james-backends-es</artifactId>
         </dependency>
         <dependency>
             <groupId>${james.groupId}</groupId>
-            <artifactId>apache-james-backends-es-v7</artifactId>
+            <artifactId>apache-james-backends-es</artifactId>
```

Comment:
Why going back to ES6 in those bindings?

Reply:
it was just to get the project compiling. For now i just deleted those modules.

Reply:
Ok yes I realized I reviewed while you were pushing latest commits, sorry :)